tests/framework/db/schema/CMssqlCommandBuilderTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+class CMssqlCommandBuilderTest extends CTestCase
+{
+	/**
+	 * @var CDbConnection
+	 */
+	private $db;
+
+	/**
+	 * Mock a CDbConection, with a CMssqlSchema and CMssqlTableSchema
+	 */
+	public function setUp()
+	{
+		/*
+		 * Disable the constructor and mock `open` so that CDbConnection does not
+		 * try to make a connection
+		 */
+		$this->db = $this->getMockBuilder(CDbConnection::class)
+			->disableOriginalConstructor()
+			->setMethods(['open', 'getServerVersion', 'getSchema'])
+			->getMock();
+
+		$schema = $this->getMockBuilder(CMssqlSchema::class)
+			->setConstructorArgs([$this->db])
+			->setMethods(['getTable'])
+			->getMock();
+
+		$tableMetaData = new CMssqlTableSchema();
+		$tableMetaData->schemaName = 'posts';
+		$tableMetaData->rawName = '[dbo].[posts]';
+		$tableMetaData->primaryKey = 'id';
+
+		$schema->method('getTable')->willReturn($tableMetaData);
+
+		$this->db->method('getSchema')->willReturn($schema);
+	}
+
+	public function testCommandBuilderOldMssql()
+	{
+		$this->db->method('getServerVersion')->willReturn('10');
+
+		$command = $this->createFindCommand([
+			'select'=>'id, title',
+			'order'=>'title',
+			'limit'=>2,
+			'offset'=>3
+		]);
+
```

Comment:
Nitpick: in this file, you could omit the newlines between $command and $this->assert to make it clear those belong together.